### PR TITLE
updated room scheme & added removeUserFromRoom in the roomcontroller

### PR DESCRIPTION
**Pull request by criticic**
Issue: #9 

#### Changes proposed in this pull request and/or Screenshots of changes:

- updates room schema
- added removeUserFromRoom & updateRoom function the room controller
- added the api/v1/room/user/delete route
- added api/v1/room/update route

(have not tested the changes yet, will do as soon as schema change is confirmed)

### CODE REVIEW

**Comment by criticic** [comment]
ill fix the merge conflict in a moment

**Comment by VatsalBhuva11** [comment]
@criticic Also, your schema is correct. Just add the schema fields from the previous merge. Then push it, test the controller, and attach the screenshots please.

File: src/controllers/room.controllers.js [hunk 1]
```diff
@@ -52,4 +52,73 @@ async function createRoom(req, res) {
     }
 }
 
-export { createRoom };
+async function removeUserFromRoom(req, res) {
+    try {
+        const { roomId} = req.body;
```
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js]: the issue stated that the `userId` for the user to be deleted will be obtained as a query param. the creator of the room (there should be a check for this too) will then disconnect the user from that room.

File: src/controllers/room.controllers.js
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js:1]: you also have to create a controller to update the room information (the name and the description). `name` and `description` may be obtained from the `body`, and the `roomId` from `query params`. it should be a `PATCH` request.
route should be: /api/v1/room/update/

File: src/controllers/room.controllers.js [hunk 2]
```diff
@@ -52,4 +52,73 @@ async function createRoom(req, res) {
...
+        }
+
+        const updatedRoom = await prisma.room.update({
+            where: {
+                id: roomId
+            },
+            data: {
+                users: {
+                    disconnect: {
+                        id: userId
```
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js]: you've created a controller to disconnect the present user from the room. while this is also a feature, it is planned to be implemented later once a certain checkpoint is reached, as it requires more than just updating the db.

Comment (criticic) [reply src/controllers/room.controllers.js]: okay no I got confused between the sender user and the user who we are removing

Comment (criticic) [reply src/controllers/room.controllers.js]: wip rn wait for 10mins have some changes im gonna push

**Comment by VatsalBhuva11** [comment]
Good job!

**Pull request merged**